oemof_b3/model/foreign_keys_update.yml

```diff
@@ -1,6 +1,8 @@
 'bus': [
     'ch4-demand',
     'ch4-shortage',
```

Comment:
Is ch4-excess missing?

Reply:
It is already added in  `foreign_keys` of `oemoflex`